### PR TITLE
Add some missing vdev properties

### PR DESCRIPTION
**Pull request by don-brady**
### Motivation and Context
I noticed that the slow_ios vdev counter was not being exposed as a vdev property like is done for the other vdev error counters (read, write, and checksum). 

### Description
This adds support for the following new read-only vdev properties: 
```
VDEV_PROP_TRIM_SUPPORT 
VDEV_PROP_TRIM_ERRORS 
VDEV_PROP_SLOW_IOS 
```

Sponsored-by: Klara, Inc. 
Sponsored-By: Wasabi Technology, Inc. 

### How Has This Been Tested?

1. Manual testing to confirm the new properties are present 
2. Ran the ZTS for zpool_get which exercises getting the vdev properties. 
```
$ zpool get trim_support,trim_errors,slow_ios dozer sda
NAME       PROPERTY      VALUE    SOURCE
sda        trim_support  on       -
sda        trim_errors   0        -
sda        slow_ios      10       -
```

### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Performance enhancement (non-breaking change which improves efficiency)
- [ ] Code cleanup (non-breaking change which makes code smaller or more readable)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Library ABI change (libzfs, libzfs\_core, libnvpair, libuutil and libzfsbootenv)
- [ ] Documentation (a change to man pages or other documentation)

### Checklist:
- [x] My code follows the OpenZFS [code style requirements](https://github.com/openzfs/zfs/blob/master/.github/CONTRIBUTING.md#coding-conventions).
- [x] I have updated the documentation accordingly.
- [x] I have read the [**contributing** document](https://github.com/openzfs/zfs/blob/master/.github/CONTRIBUTING.md).
- [ ] I have added [tests](https://github.com/openzfs/zfs/tree/master/tests) to cover my changes.
- [ ] I have run the ZFS Test Suite with this change applied.
- [x] All commit messages are properly formatted and contain [`Signed-off-by`](https://github.com/openzfs/zfs/blob/master/.github/CONTRIBUTING.md#signed-off-by).

### CODE REVIEW

File: module/zcommon/zpool_prop.c [hunk 1]
```diff
@@ -381,6 +381,12 @@ vdev_prop_init(void)
 	zprop_register_number(VDEV_PROP_INITIALIZE_ERRORS,
 	    "initialize_errors", 0, PROP_READONLY, ZFS_TYPE_VDEV, "<errors>",
 	    "INITERR", B_FALSE, sfeatures);
+	zprop_register_number(VDEV_PROP_TRIM_ERRORS, "trim_errors", 0,
+	    PROP_READONLY, ZFS_TYPE_VDEV, "<errors>", "TRERR", B_FALSE,
```
Comment (amotin) [inline module/zcommon/zpool_prop.c:385]: Wouldn't `TRIMERR` be more consistent with `INITERR` above and `TRIMSUP` below?

**Review by amotin — APPROVED** [review]
Looks good to me aside of one thought:

File: module/zcommon/zpool_prop.c (hunk 1 above)
Comment (don-brady) [reply module/zcommon/zpool_prop.c]: yes, I'll make the change. Thanks.

**Review by tonyhutter — APPROVED** [review]

**Comment by tonyhutter** [comment]
@don-brady any last changes or is this ready for merge?

**Comment by don-brady** [comment]
> @don-brady any last changes or is this ready for merge?

@tonyhutter  It's ready.  Thanks

**Pull request merged**